windse/SolverManager.py

```diff
@@ -238,6 +238,15 @@ def EvaluateObjective(self):
 
         annotate = self.params.dolfin_adjoint 
 
+        ### Convert to list ###
+        if isinstance(self.objective_type,str):
+            self.objective_type = {self.objective_type: obj_funcs.objective_kwargs[self.objective_type]}
+        elif isinstance(self.objective_type,list):
+            new_objective_type = {}
+            for obj in self.objective_type:
+                new_objective_type[obj] = obj_funcs.objective_kwargs[obj]
+            self.objective_type = new_objective_type
+
```

Comment:
@jefalon Is this code now deprecated by what happens in ParameterManager?

Reply:
i think that can be deleted